infra/embedding_step_functions/unified_embedding/handlers/split_into_chunks.py

```
@@ -17,22 +24,94 @@
+    else:
+        return _split_into_chunks(event)
+
+
+def _load_chunks_from_s3(event: Dict[str, Any]) -> Dict[str, Any]:
+    """Load chunks from S3 and return them inline."""
+    chunks_s3_key = event.get("chunks_s3_key")
+    chunks_s3_bucket = event.get("chunks_s3_bucket")
+    batch_id = event.get("batch_id")
+
+    if not chunks_s3_key or not chunks_s3_bucket:
+        raise ValueError("chunks_s3_key and chunks_s3_bucket are required for load_chunks_from_s3 operation")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)